src/main/java/com/o19s/es/ltr/feature/store/StoredLtrModel.java

```diff
@@ -262,6 +294,10 @@ public String getType() {
             return type;
         }
 
+        public void setNamedFeatureNormalizers(List<FeatureNormDefinition> featureNormalizers) {
+            this.featureNormalizerSet.addFeatureNormalizers(featureNormalizers);
```

Comment:
instead of adding I that it should set a this.featureNormalizerSet = new StoredFeatureNormalizers(featureNormalizers);